lib/SPIRV/libSPIRV/SPIRVEntry.cpp

```diff
@@ -388,11 +388,11 @@ SPIRVEntry::setLinkageType(SPIRVLinkageTypeKind LT) {
   addDecorate(new SPIRVDecorateLinkageAttr(this, Name, LT));
 }
 
-std::ostream &
-operator<<(std::ostream &O, const SPIRVEntry &E) {
+llvm::raw_ostream &
+operator<<(llvm::raw_ostream &O, const SPIRVEntry &E) {
   E.validate();
   E.encodeAll(O);
-  O << SPIRVNL;
+  SPIRVNL(O);

```

Comment:
SPIRVNL is defined as a ostream manipulator. spv_ostream can support the same syntax by defining a << operator accepting a manipulator, e.g.

typedef spv_ostream& (*spv_ostream_manip_t)(spv_ostream &IS);

spv_ostream& operator<<(spv_ostream& O, spv_ostream_manip_t M) {
  M(O);
  return O;
}